frontend/src/cards/AgeAdjustedTableCard.tsx

```diff
@@ -92,8 +89,9 @@ export function AgeAdjustedTableCard(props: AgeAdjustedTableCardProps) {
     config.metricId.includes("ratio")
   );
 
-  const cardTitle = `${metrics[0]?.fullCardTitleName
-    } in ${props.fips.getSentenceDisplayName()}`;
+  const cardTitle = `${
+    metrics[0]?.fullCardTitleName
```

Comment:
is there a way to make the 0 index here more semantic? (destructure and name a variable for the metric?)